Source/Urho3D/UI/UIElement.cpp

```diff
@@ -418,31 +418,31 @@ const IntVector2& UIElement::GetScreenPosition() const
     return screenPosition_;
 }
 
-void UIElement::OnHover(const IntVector2& position, const IntVector2& screenPosition, int buttons, int qualifiers, Cursor* cursor)
+void UIElement::OnHover(const IntVector2& position, const IntVector2& screenPosition, MouseButtonFlags buttons, QualifierFlags qualifiers, Cursor* cursor)
 {
     hovering_ = true;
 }
 
-void UIElement::OnDragBegin(const IntVector2& position, const IntVector2& screenPosition, int buttons, int qualifiers,
+void UIElement::OnDragBegin(const IntVector2& position, const IntVector2& screenPosition, MouseButtonFlags buttons, QualifierFlags qualifiers,
     Cursor* cursor)
 {
-    dragButtonCombo_ = buttons;
+    dragButtonCombo_ = buttons.AsInteger();

```

Comment:
Can `dragButtonCombo_ ` be `MouseButtonFlags` as well?

Reply:
Yes, it is possible. I changed it now.